TravelAgency-persistence/src/main/java/cz/muni/fi/travelAgency/entities/Customer.java

```diff
@@ -0,0 +1,163 @@
+package cz.muni.fi.travelAgency.entities;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+
+/**
+ * Class representing Customer entity.
+ *
+ * @author Filip Cekovsky
+ */
+@Entity
+@Table(name = "CUSTOMERS")
+public class Customer {
+    /** Unique ID set by the DB */
+    @Id
+    @GeneratedValue(strategy =  GenerationType.IDENTITY)
+    private Long id;
+
+    /** Name of the customer */
+    @NotNull
+    @Column(nullable = false)
+    private String name;
+
+    /** Surname of the customer */
+    @NotNull
+    @Column(nullable = false)
+    private String surname;
+
+    /** Email address - compulsory*/
+    @NotNull
+    @Column(nullable = false)
+    private String email;
+
+    /** Nullable phone number */
+    @Column
+    private String phoneNumber;
+
+    /** Passport number */
+    @Column
+    private String passportNumber;
```

Comment:
Passport number should be unique because there can't be two customers with the same passport number.